domain/lib/src/usecases/myspace/copy_multiple_files_to_my_space_interactor.dart

```diff
@@ -34,25 +34,32 @@ import 'package:dartz/dartz.dart';
 import 'package:domain/domain.dart';
 import 'package:flutter/material.dart';
 
-class CopyMultipleToMySpaceInteractor {
+class CopyMultipleFilesToMySpaceInteractor {
   final CopyToMySpaceInteractor _copyToMySpaceInteractor;
 
-  CopyMultipleToMySpaceInteractor(this._copyToMySpaceInteractor);
+  CopyMultipleFilesToMySpaceInteractor(this._copyToMySpaceInteractor);
 
-  Future<Either<Failure, Success>> execute({@required List<CopyRequest> copyRequests}) async {
-    final listResult = await Future.wait(copyRequests.map((element) =>
+  Future<Either<Failure, Success>> execute({@required List<WorkGroupNode> workGroupNodes}) async {
+    if (workGroupNodes.every((element) => element is WorkGroupFolder)) {
```

Comment:
IMO, folders are not support in `Copy to MySpace`. `every` should be `any`

Reply:
please refer in: https://github.com/linagora/linshare-mobile-flutter-app/issues/182#issuecomment-781771402